t/13-joblocks.t

```diff
@@ -29,11 +29,7 @@ $t->post_ok('/api/v1/mutex', form => {name => 'test_lock'})->status_is(403);
 $t->post_ok('/api/v1/mutex/test_lock', form => {action => 'lock'})->status_is(403);
 $t->post_ok('/api/v1/mutex/test_lock', form => {action => 'unlock'})->status_is(403);
 
-$t->ua->on(
-    start => sub {
-        my ($ua, $tx) = @_;
-        $tx->req->headers->add('X-API-JobToken' => $tokenA);
-    });
+$t->ua->on(start => sub ($ua, $tx) { $tx->req->headers->add('X-API-JobToken' => $tokenA) });
```

Comment:
Just a note because I read a dicussion on p5p the other day:
It could be wise to always add a `, @` to the signature of third-party callbacks. They might add arguments in the future.

Reply:
That's a good point. Let's try to remember that.